### PR TITLE
Translation of text into Urdu language

### PR DESCRIPTION
**Pull request by arsheelsheikh**
Added Urdu language with changes in ur.json file

Translation of text into Urdu language 

This PR issue is related to https://github.com/appwrite/appwrite/issues/1724#issuecomment-935586036

### CODE REVIEW

**Comment by arsheelsheikh** [comment]
@christyjacob4 I have made new PR modifying changes in ur.jason file

File: app/config/locale/translations/ur.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.magicSession.footer": "اگر آپ نے اس ای میل کا استعمال کرتے ہوئے لاگ ان کرنے کے لیے نہیں کہا تو آپ اس پیغام کو نظر انداز کر سکتے ہیں۔",
+    "emails.magicSession.thanks": "شکریہ",
+    "emails.magicSession.signature": "ٹیم۔ {{project}}",
+    "emails.recovery.subject": "پاس ورڈ ری سیٹ۔",
+    "emails.recovery.hello": "ہیلو {{name}}",
+    "emails.recovery.body": "{{project}} کا پاس ورڈ تبدیل کرنے کے لیے درج ذیل لنک پر عمل کریں",
+    "emails.recovery.footer": "اگر آپ نے اپنا پاس ورڈ دوبارہ ترتیب دینے کے لیے نہیں کہا تو آپ اس پیغام کو نظر انداز کر سکتے ہیں۔",
+    "emails.recovery.thanks": "شکریہ",
+    "emails.recovery.signature": "ٹیم۔ {{project}}",
+    "emails.invitation.subject": "٪ s پر٪ s ٹیم کو دعوت",
```
Comment (christyjacob4) [inline app/config/locale/translations/ur.json:24]: You need to use the right `%s` characters here

File: app/config/locale/translations/ur.json [hunk 2]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.invitation.hello": "خوش آمدید",
+    "emails.invitation.body": "یہ پیغام آپ کو اس لیے بھیجا گیا تھا کہ {{مالک}} نے آپ کو {{project}} میں {{team}} ٹیم کا رکن بننے کی دعوت بھیجی",
```
Comment (christyjacob4) [inline app/config/locale/translations/ur.json:26]: You need to keep the words inside the curly braces as is. 
`{{مالک}}` this needs to be fixed

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/ur.json (hunk 1 above)
Comment (arsheelsheikh) [reply app/config/locale/translations/ur.json]: thank you for pointing this out

File: app/config/locale/translations/ur.json (hunk 2 above)
Comment (arsheelsheikh) [reply app/config/locale/translations/ur.json]: @christyjacob4 Done

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Comment by arsheelsheikh** [comment]
@christyjacob4 kindly check your discord dm, I have already sent you a message.

**Pull request merged**